trillian/ctfe/handlers_test.go

```diff
@@ -2205,3 +2330,13 @@ func loadCertsIntoPoolOrDie(t *testing.T, certs []string) *PEMCertPool {
 	}
 	return pool
 }
+
+func mustMarshalRoot(lr *types.LogRootV1) *trillian.SignedLogRoot {
+	rootBytes, err := lr.MarshalBinary()
+	if err != nil {
+		glog.Fatalf("Failed to marshal root in test: %v", err)
```

Comment:
Might be better to have this take a `t *testing.T` first parameter, then call `t.Helper()` and use `t.Fatalf`

Reply:
Done.

Reply:
Ah that breaks `sth_test` because I shared test cases for log / mirror and there's no `testing.T` when they're declared.

Reply:
Should be fixed now.